fleet_vehicle_fuel_capacity/models/fleet_vehicle.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 - TODAY, Marcel Savegnago
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class FleetVehicle(models.Model):
+
+    _inherit = 'fleet.vehicle'
+
+    fuel_capacity = fields.Float(
+        string='Fuel Capacity(l)',
```

Comment:
```suggestion
        string='Fuel Capacity',
```

What is this "(l)" for? Is it for litres?
If so please add a space

```suggestion
        string='Fuel Capacity (L)',
```

Alternatively you could add this directly in the view using `oe_inline`, like this: https://github.com/odoo/odoo/blob/084d06a5892ab2723d02dc29f0e62377c554efac/addons/product_expiry/views/product_template_views.xml#L15-L19

it looks slightly better ;)